### ISSUE TITLE
please publish to npm registry

### ISSUE THREAD

**Issue by ctaepper** [dibari/angular-ellipsis#57]
there is allready a package called angular-ellipsis on npm, which is a fork of this project... however, its an old version and doesnt seem to be maintained

as a browserify/webpack user i manage my dependencies via npm and try not to use bower

**Comment by Duder-onomy** [comment]
Agreed. I thought I was losing my mind when the repo with this name on NPM didn't work at all. 

Please publish to NPM so we can use this without any hacks.

**Comment by mattjanssen** [comment]
:+1:

**Comment by andyskw** [comment]
It would be really great!

**Comment by andyskw** [comment]
Hey @dibari, do you think it would be possible to get the package published to npm as well? what do you think, how much effort would it be to regain the ownership over the original angular-ellipsis package? Is there anything the community could do for you about it?

**Comment by ventralnet** [comment]
+1

**Comment by ventralnet** [comment]
I emailed npm support for what happens when someone hijacks a project and the response is below...  It looks like it will take some communication with @dibari to get this published to npm.  Or @dibari can change the package name which would probably be the most immediate solution to this (maybe angular-ellipsis-original or something).  Either way this would really help our webpack users.

```
Thanks for emailing. In this case, we'd recommend the package dispute process. We wouldn't remove this package on the basis of being a fork.

https://docs.npmjs.com/policies/disputes

Essentially, email the current owner and cc support@npmjs.com, with your request to take over the package. We will help with the discussion as needed. Our primary goal is that you and the other user can (hopefully) resolve this respectfully between the two of you.

Let me know if you have any further questions, I'll be here to help.
```
